preferences.json

```diff
@@ -7,7 +7,7 @@
       "y" : 23
     }
   },
-  "contactListFilePath" : "data/contactlist.json",
-  "taskListFilePath" : "data/tasklist.json",
-  "moduleListFilePath" : "data/modulelist.json"
+  "contactListFilePath" : "data\\contactlist.json",
```

Comment:
I think there is no need to push the preferences.json?

Reply:
Fixed!